MAUI/Scheduler/accessibility.md

```diff
@@ -0,0 +1,231 @@
+<tr>
+<td>
+Enter<br/><br/></td><td>
+It triggers Tapped event<br/><br/></td></tr>
+</table>
+
+### Month view
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Right arrow<br/><br/></td><td>Moves selection to the next (LTR) or previous (LTR) date of the currently selected date.<br/><br/></td></tr>
+<tr>
+<td>
+Left arrow<br/><br/></td><td>
+Moves selection to the previous (LTR) or next (RTL) date of the currently selected date. <br/><br/></td></tr>
+<tr>
+<td>
+Down arrow<br/><br/></td><td>
+Moves selection to the date directly below the currently selected date on the next row.<br/><br/></td></tr>
+<tr>
+<td>
+Up arrow<br/><br/></td><td>
+Moves selection to the date directly above the currently selected date on the previous row.<br/><br/></td></tr>
+<tr>
+<td>
+Enter<br/><br/></td><td>
+It triggers Tapped event<br/><br/></td></tr>
+</table>
+
+### Appointments
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td>Moves selection to the next (LTR) or Previous (RTL) appointment of the currently selected appointment. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Moves selection to the previous (LTR) or next (RTL) appointment of the currently selected appointment. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Delete<br/><br/></td><td>Deletes the selected appointment from appointments collection.<br/><br/></td></tr>
+<tr>
+<td>
+Enter<br/><br/></td><td>
+It triggers Tapped event<br/><br/></td></tr>
+</table>
+
+### View navigations
+
+#### Windows
+
+<table>
+<tr>
+<td>
+Ctrl + Alt + 1 => DayView<br/><br/></td><td>
+Moves the view to day view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 2 => WeekView<br/><br/></td><td>
+Moves the view to week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 3 => WorkWeekView<br/><br/></td><td>
+Moves the view to work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 4 => MonthView<br/><br/></td><td>
+Moves the view to work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 5 => TimelineDayView<br/><br/></td><td>
+Moves the view to Timeline day view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 6 => TimelineWeekView<br/><br/></td><td>
+Moves the view to Timeline week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 7 => TimelineWorkWeekView<br/><br/></td><td>
+Moves the view to Timeline work week view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 8 => TimelineMonthView<br/><br/></td><td>
+Moves the view to Timeline month view.<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Alt + 9 => AgendaView<br/><br/></td><td>
+Moves the view to Agenda view.<br/><br/></td></tr>
+</table>
+
+N> There is a framework issue with unable to remove or modify  default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193).
```

Comment:
N> There is a framework issue with unable to remove or modify the default visual focus style on tab press in windows and the following framework issue in this report [link](https://github.com/dotnet/maui/issues/10193):